src/runtime/c_runtime_api.cc

```diff
@@ -485,7 +486,7 @@ int TVMFuncCreateFromCFunc(TVMPackedCFunc func, void* resource_handle, TVMPacked
       int ret = func(const_cast<TVMValue*>(args.values), const_cast<int*>(args.type_codes),
                      args.num_args, rv, rpack.get());
       if (ret != 0) {
-        throw dmlc::Error(TVMGetLastError() + ::dmlc::StackTrace());
+        throw tvm::Error(TVMGetLastError() + tvm::Backtrace());
```

Comment:
if it is in runtime namespace, the file should be in runtime namespace, then exposed to the tvm namespace